@here/harp-test-utils/lib/ProfileHelper.ts

```diff
@@ -11,14 +11,15 @@ import * as fs from "fs";
 //
 // `performance`, `PerformanceObserver`, ... are global in Browser environment, but only available
 // from `perf_hooks` in node env.
-//
-if (typeof window === "undefined") {
-    const perfHooks = require("perf_hooks");
-
-    (global as any).performance = perfHooks.performance;
-    (global as any).PerformanceObserver = perfHooks.PerformanceObserver;
-    (global as any).PerformanceEntry = perfHooks.PerformanceEntry;
-}
+// Note, the following had to be commented out, because "karma-typescript" tries to resolve the
```

Comment:
can it be removed at once?

Also, why it doesn't apply to the same code in
`test/performance/LinesPerformanceTest.ts` and
`test/performance/OmvDecoderPerformanceTest.ts`?

Reply:
discussed internally - it will be taken care of in a separate ticket.